### PR TITLE
completed html-forms

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </div>
+    <div>
+      <p>Favorite Language (choose one)</p>
+      <label for="favorite-language"></label>
+      <div>
+        <input type="radio" id="html" name="language" value="html"> HTML
```
Comment (Zhiwutian) [inline html-forms/index.html:32]: make sure you have a label for each radio input, you can wrap them around the input if desired. That will maintain the same functionality

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good, small update needed!

**Comment by m-Cho1** [comment]
image
Edited!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**